source/geometry_model/sphere.cc

```diff
@@ -38,8 +38,13 @@ namespace aspect
       GridGenerator::hyper_ball (coarse_grid,
                                  Point<dim>(),
                                  R);
+
+#if DEAL_II_VERSION_GTE(9,0,0)
+      coarse_grid.set_manifold(0,spherical_manifold);
```

Comment:
But doesn't this imply setting it for all objects? Wouldn't you have to give a different manifold id to only the faces at the boundary, and then attach the manifold to the triangulation for that specific manifold id?

I have to admit that I don't quite recall what the default manifold id of objects is. Your code seems to suggest that it is something different than zero, and that for reasons I don't recall, `GridGenerator::hyper_ball()` only sets the manifold id of boundary faces to zero. Is this correct?

Reply:
> But doesn't this imply setting it for all objects? 

Yes. The default manifold id is 0 for every object and currently no ``GridGenerator`` function touches the manifold id.